python/pyglome/glome.py

```diff
@@ -114,7 +114,8 @@ def __init__(self,
               Must be an integer in range 1-32.
         Raises:
             ValueError: Raised whenever min_peer_tag_len is not in
-              range 1-32.
+              range 1-32. Also when null shared secret is derived from
+              user/peer key pair.
```

Comment:
When does that happen? If it is the .exchange() function that returns null, consider using a specific error such as ExchangeError or something and actively check for None.

Reply:
Is the exchange function who raises ValueError exception when some public keys are provided. WDYT about `raise ExchangeError from ValueError`?